skype/skype-ps/skype/New-CsHuntGroup.md

```diff
@@ -89,44 +85,45 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+## OPTIONAL PARAMETERS
+
 ### -AgentAlertTime
-PARAMVALUE: Int16
+The AgentAlertTime parameter represents the time (in seconds) that a call can remain unanswered before it is automatically routed to the next agent. The AgentAlertTime can be set to any integer value between 30 and 180 seconds (3 minutes), inclusive. The default value is 30 seconds.
 
 ```yaml
-Type: Object
+Type: Int16
 Parameter Sets: (All)
 Aliases: 
 Applicable: Skype for Business Online
 
 Required: False
 Position: Named
-Default value: None
+Default value: 30
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-
-### -BypassDualWrite
-PARAMVALUE: $true | $false
+### -AllowOptOut
+The AllowOptOut parameter indicates whether or not agents can opt in or opt out from taking calls from a hunt group.
 
 ```yaml
-Type: Object
+Type: System.Boolean
 Parameter Sets: (All)
 Aliases: 
 Applicable: Skype for Business Online
 
 Required: False
 Position: Named
-Default value: None
+Default value: True
 Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ClientAudience
-PARAMVALUE: String
+### -DistributionLists
+The DistributionLists parameter lets you add all the members of the distribution lists to the hunt group. This is a list of distribution list GUID. A service wide configurable maximum number of DLs per HuntGroup are allowed. Only the first N (service wide configurable) agents from all distribution lists combined are considered for accepting the call. Nested DLs are supported. O365 Groups can also be used to add members to the hunt group.
```

Comment:
GUID -> GUIDs

Reply:
also, lets you add **all** the members,  we can remove "add all"

Reply:
Fixed